cratejoy_api_examples/generic_get.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/local/bin/python
```

Comment:
Oh, the way you wrote it is fine :) Just a good habit that makes it slightly easier to upgrade Python someday!